ops_boba/cloudformation/11-infrastructure-transaction-service-gnosis-mainnet.yaml

```diff
@@ -0,0 +1,181 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "Gnosis-safe Application Infrastructure"
+
+Parameters:
+
+  EcsImageId:
+    # TODO: Implement automatic update or notifications: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS-AMI-SubscribeTopic.html
+    Description: Image ID for running the ECS Cluster instances
+    Type: AWS::SSM::Parameter::Value<AWS::EC2::Image::Id>
+    Default: /aws/service/ecs/optimized-ami/amazon-linux-2/recommended/image_id
+
+  EcsInstanceType:
+    # https://aws.amazon.com/ec2/instance-types/
+    # The types of instances is limited to avoid mistakes generating huge costs by putting the wrong instance type.
+    Description: Type of instances to be added to the ECS cluster
+    Type: String
+    Default: t3a.xlarge
+
+  InfrastructureStackName:
+    Description: Infrastructure stack to associate this stack with
+    Type: String
+    Default: infrastructure-core
+
+Resources:
+
+  LoadBalancerGnosisSecurityGroup:
```

Comment:
This is sort of confusing because Network Load Balancers don't have Security Groups, so whatever is going to use this SG isn't going to be the Load Balancer?

Reply:
I don't think this `LoadBalancerGnosisSecurityGroup` is consumed anywhere so it should be safe to remove it.

Reply:
This Security group might be named a bit off - but it's used in the application stack network setup
```
SecurityGroups:
            - Fn::ImportValue: !Sub "${InfrastructureStackName}:LoadBalancerGnosis:SecurityGroup"
```

to be specific, here: https://github.com/omgnetwork/optimism-v2/pull/288/files#diff-50da5576a6644e3332ae77ad1a759ee51feecf3c915eb3098668648367a386b4R387-R391

Reply:
I presume the problem is the naming right - NetworkSecurityGroup perhaps?